anima/env/resolve/conformer.py

```diff
@@ -3,3 +3,201 @@
 #
 # This module is part of anima and is released under the MIT
 # License: http://www.opensource.org/licenses/MIT
+
+import os
+import re
+import glob
+from stalker import Project, Task
+
+from anima.ui.base import AnimaDialogBase, ui_caller
+from anima.ui.lib import QtCore, QtGui, QtWidgets
+from anima.env import blackmagic
+from anima.utils import do_db_setup
+
+do_db_setup()
```

Comment:
This should not be here, right? It will be called every time you import ``conformer`` and if this is not going to imported you should use:

```python
if __name__ == '__main__':
    do_db_setup()
```

at the end of this file.